thirdeye-server/src/main/java/ai/startree/thirdeye/core/ExceptionHandler.java

```diff
@@ -92,9 +92,5 @@ private static void populateStatusListApi(final Throwable e, StatusListApi statu
         break;
       }
     }
-
-    if (e.getCause() != null) {
```

Comment:
This actually helps understand the cause. Else, we have to see the logs. We don't have an alternate mechanism

Reply:
Ok I'll leave them in. I was going to do one big ui alert that contains all the messages, but the display looks messy so I'll leave it as is.